desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml

```diff
@@ -20,12 +31,19 @@
             the security pitfalls that Flash had a reputation for.
             Ruffle puts Flash back on the web, where it belongs!
         </p>
+        <p xml:lang="fr">Ruffle est un émulateur pour Flash Player écrit en Rust. En tirant parti des garanties de sécurité offertes par Rust, nous pouvons éviter tous les problèmes de sécurité pour lesquels Flash était réputé. Ruffle remet Flash à sa juste place, sur Internet !</p>
+        <p xml:lang="nl">Ruffle is een Flash speler geschreven in Rust. Door gebruik te maken van de veiligheidsgaranties die Rust biedt, kunnen we zelfverzekerd de veiligheidsproblemen waar Flash een reputatie voor had vermijden. Ruffle zet Flash terug op het web, waar het thuis hoort!</p>
         <p xml:lang="pl">Ruffle jest emulatorem Flash Playera napisanym w języku programowania Rust. Wykorzystując gwarancje bezpieczeństwa języka Rust, możemy pewnie uniknąć wszystkich luk bezpieczeństwa, z których Flash był znany. Ruffle przywraca Flasha do sieci, gdzie jest jego miejsce!</p>
+        <p xml:lang="ru">Ruffle — это эмулятор Flash Player'а, написанный на Rust. Опираясь на гарантии безопасности Rust, мы можем с уверенностью избежать всех проблем с безопасностью, которыми был известен Flash. Ruffle возращает Flash в Сеть на своё прежнее место!</p>
```

Comment:
That differs from the following, but probably should be similar? Mentioning it because one translation might be better than the other one.

https://github.com/ruffle-rs/ruffle/blob/35f1fda4b2eb363696842b4d4fc23a9ad8cf0e0f/web/packages/extension/assets/_locales/ru/messages.json#L93

Reply:
Never mind, that got changed there in this PR too...

Reply:
Actually... There seems to be a typo: `Возращает` vs `Возвращает`

Reply:
Good catch! Fixed on Crowdin.